content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-disable-policy-enforcement.md

```diff
@@ -105,4 +105,12 @@ spec:
       }
 ```
 
-Again, the `mode` configuration is set to `warn`. If you are logged into the Enforce Console, you can access the [Disallow privilege escalation policy](https://console.enforce.dev/policies/catalog/create/disallow-privilege-escalation-cue) from the Policy Catalog from the **Create policy** button.
\ No newline at end of file
+Again, the `mode` configuration is set to `warn`. If you are logged into the Enforce Console, you can access the [Disallow privilege escalation policy](https://console.enforce.dev/policies/catalog/create/disallow-privilege-escalation-cue) from the Policy Catalog from the **Create policy** button.
+
+## Review Warning Policies
+
+You can review which policies are in **Enforce** or **Warn** mode by reviewing the policy table in your Enforce Console.
+
+![Warn or Enforce mode indicated in policy table](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/enforce-console-warn-enforce-mode_hu3b0eaea9ffe2caf7848b0a93077fa970_51901_900x0_resize_box_3.png)
```

Comment:
We'll need to merge this before we can ensure that this works, since I thought it would be better to just upload the image once. Not sure about that resize bit, that is what seems to be showing up if you click on image source.